lib/rucio/common/rse_attributes.py

```diff
@@ -1,13 +1,25 @@
-# Copyright European Organization for Nuclear Research (CERN)
+# -*- coding: utf-8 -*-
```

Comment:
what's this line?

Reply:
Setting the encoding of the file to utf-8 for Python 2. Python 3 already has this as a default. It allows special characters in the file. I added it to the add_header script because some author names had special characters and that's now fine with this.

Reply:
hmmmm, not sure about this... i'd rather not add anything for python2 at this stage anymore. @bari12 ?

Reply:
I would leave it, probably even post 1.24. We do still keep Python 2.7 compatibility for the clients, so in theory it might be needed there and doesn't hurt really. What I would remove though, at least with 1.24 is the `# PY3k Compatible` line.

Reply:
Funny that you mention the `# PY3K COMPATIBLE` line, because I removed that from the add_header script along with the python3 test suite in my Python 3.7 branch :)